webpack.config.production.js

```diff
@@ -15,7 +15,7 @@ config.entry = './app/index';
 
 config.output.publicPath = '../dist/';
 
-config.module.loaders.push({
+config.module.loaders = config.module.loaders.concat([{
```

Comment:
Looks `.production.js` and `.electron.js` will have problem of object reference in `package.js`, so I handle these with `array.concat`.